resources/wrapper/godelw

```diff
@@ -161,23 +164,35 @@ GODEL_BASE_DIR=${GODEL_HOME:-$HOME/.godel}
 # determine OS
 OS=""
 EXPECTED_CHECKSUM=""
-case "$(uname)" in
-    Darwin*)
+case "$(uname)-$(uname -m)" in
+    Darwin-x86_64)
         OS=darwin
-        EXPECTED_CHECKSUM=$DARWIN_CHECKSUM
+        ARCH=amd64
+        EXPECTED_CHECKSUM=$DARWIN_AMD64_CHECKSUM
         ;;
-    Linux*)
+    Darwin-aarch64)
```

Comment:
This should be `Darwin-arm64` (verified locally that output of `uname -m` on an M1 mac is `arm64`)

Reply:
made this change